paper-select.js

```diff
@@ -423,8 +431,19 @@ Polymer({
     if (this.multiple) {
       if (!this.bindValue)
         this.set('bindValue', [item]);
-      else
-        this.push('bindValue', item);
+      else {
+        if(this.unique) {
+          var self = this;
+          var existItems = this.bindValue.filter(function(addedItem) {
+            if(!addedItem[self.uniqueField] && !item[self.uniqueField])
+              return false;
+            return addedItem[self.uniqueField].toLowerCase().indexOf(item[self.uniqueField]) === 0;
+          });
+          if(existItems.length == 0)
```

Comment:
If item is not found from exist items then only it will add new item otherwise it will ignore.